create_dev_meetings_report.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
A small description of the code might be great, as well in the other python script

Reply:
something like this at the top of the create_dev_meetings_reports.py ? :  """
this script will run once a week (on Friday at 6 am? Time will be defined in the dev_meeting_reports.yml
- cron syntax) to create a Gammapy Developer Meeting Report containing pull requests which have been
opened or merged as well as issues opened in the last 7 (8?)  days
"""